File: dotnet/src/Pactman.NonprofitCheckPlus/Endpoints.cs
namespace Pactman.NonprofitCheckPlus
{
    /// <summary>The API surface this SDK wraps, declared once.</summary>
    public static class Endpoints
    {
        /// <summary>
        /// Path of the single-check endpoint. <c>{ein}</c> is replaced with a normalized EIN.
        /// </summary>
        public const string SingleCheckPath = "/api/entities/nonprofitcheck/v1/us/ein/{ein}";

        /// <summary>Path of the bulk-check endpoint.</summary>
        public const string BulkCheckPath = "/api/entities/nonprofitcheckbulk/v1/us/eins";

        /// <summary>
        /// Maximum number of EINs the API accepts in one bulk request.
        /// </summary>
        /// <remarks>This mirrors the server-side limit. It is declared once, here.</remarks>
        public const int MaxBulkEins = 50;
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/Endpoints.java
package org.pactman.nonprofitcheckplus;

/** The API's paths and limits, declared once. */
public final class Endpoints {

    /**
     * Path of the single-check endpoint.
     *
     * <p>{@code {ein}} is replaced with a normalized EIN.
     */
    public static final String SINGLE_CHECK_PATH = "/api/entities/nonprofitcheck/v1/us/ein/{ein}";

    /** Path of the bulk-check endpoint. */
    public static final String BULK_CHECK_PATH = "/api/entities/nonprofitcheckbulk/v1/us/eins";

    /**
     * Maximum number of EINs the API accepts in one bulk request.
     *
     * <p>This mirrors the server-side limit. It is declared once, here.
     */
    public static final int MAX_BULK_EINS = 50;

    private Endpoints() {
    }
}
